stacks-signer/src/config.rs

```diff
@@ -141,6 +142,9 @@ pub struct SignerConfig {
     pub tenure_idle_timeout: Duration,
     /// The maximum age of a block proposal in seconds that will be processed by the signer
     pub block_proposal_max_age_secs: u64,
+    /// Time following a block's global acceptance that a signer will consider an attempt by a miner to reorg the block
+    /// as valid towards miner activity
```

Comment:
Can you clarify here that this only applies to the first block in a tenure reorging the last block(s) in the previous tenure?